### PR TITLE
IG Importer doesn't load

### PR DESCRIPTION
**Pull request by drizzentic**
Bug fixes

## Summary by CodeRabbit

- **New Features**
  - Updated configuration files to include new Kafka topics, OpenHIM settings, and service URLs.
  - Added new environment variables for OpenHIM API host and port.

- **Improvements**
  - Enhanced URL handling for the FHIR IG microfrontend app by switching from local to cloud-based URLs.
  - Improved CORS handling for `OPTIONS` requests in Nginx reverse proxy configuration.

- **Configuration Changes**
  - Adjusted `.env` files to include various service endpoints and ports.
  - Updated `CLUSTERED_MODE` setting in `.env.remote` for optimized system integration and access control.

### CODE REVIEW

**Comment by rcrichton** [comment]
Task linked: [CU-86byxfvwm IG Importer doesn't load](https://app.clickup.com/t/86byxfvwm)

File: interoperability-layer-openhim/docker-compose.yml [hunk 1]
```diff
@@ -51,6 +51,8 @@ services:
       KC_FRONTEND_URL: ${KC_FRONTEND_URL}
       OPENHIM_CONSOLE_SHOW_LOGIN: ${OPENHIM_CONSOLE_SHOW_LOGIN}
       REACT_APP_OPENHIM_API_BASE_URL: ${OPENHIM_API_BASE_URL}
+      OPENHIM_API_HOST: ${OPENHIM_API_HOST}
+      OPENHIM_API_PORT: ${OPENHIM_API_PORT}
```
[the remark this answers is not in the document]
Comment (drizzentic) [reply interoperability-layer-openhim/docker-compose.yml:55]: This is being handled by instant v2

File: .env.local [hunk 2]
```diff
@@ -24,3 +24,28 @@ JS_REPORT_PACKAGE_PATH=
...
+KC_JEMPI_ROOT_URL=https://jempi-web.platform.jembi.cloud
+KC_SUPERSET_ROOT_URL=https://superset.platform.jembi.cloud
+KC_OPENHIM_ROOT_URL=https://openhimconsole.platform.jembi.cloud
+GF_SERVER_DOMAIN=grafana.platform.jembi.cloud
+
+REACT_APP_JEMPI_BASE_API_HOST=https://jempi-api.platform.jembi.cloud
+REACT_APP_JEMPI_BASE_API_PORT=443
+OPENHIM_CONSOLE_BASE_URL=https://openhimconsole.platform.jembi.cloud
+OPENHIM_API_HOST=https://openhimcore.platform.jembi.cloud
+OPENHIM_API_PORT=443
```
Comment (bradsawadye) [inline .env.local:51]: This is for remote deployments, and should not be in the local env file which is used for deploying locally. And also change the domain name to be `OPENHIM_API_HOST=https://openhimcore.<DOMAIN_NAME>`. Implementers can then sub in their domain name when deploying

**Review by brett-onions — APPROVED** [review]

**Review by brett-onions — APPROVED** [review]

**Review by bradsawadye — APPROVED** [review]

File: fhir-ig-importer/importer/volume/ig-importer-app.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "description": "FHIR IG microfrontend app",
   "category": "HIE Configuration",
   "type": "esmodule",
-  "url": "http://localhost:3334/jembi-fhir-ig-importer.js",
+  "url": "https://openhimconsole.<domain>/fhir-ig-importer",
```
Comment (bradsawadye) [inline fhir-ig-importer/importer/volume/ig-importer-app.json:6]: can we still access this when running locally?

Comment (drizzentic) [reply fhir-ig-importer/importer/volume/ig-importer-app.json:6]: Yes, but the domain should point to the console URL. Should I modify this to `<openhim-console-url>/fhir-ig-importer` @bradsawadye? Also, I have created a ticket that will solve the issue of hardcoded properties

**Pull request merged**